ci/spack.yaml

```diff
@@ -0,0 +1,18 @@
+spack:
```

Comment:
Can you put some comments?

Also did we decide to always use yml not yaml? Or does spack want yaml?

Reply:
Will do.

Spack wants "spack.yaml"